spotbugs/src/main/java/edu/umd/cs/findbugs/config/CommandLine.java

```diff
@@ -342,9 +354,35 @@ private int parse(String argv[], boolean dryRun) throws IOException, HelpRequest
             }
         }
 
+        if (!dryRun && !parsingStdin && parseStdinAsOptions) {
+            String[] stdinArgv = readStdinAsArgv();
+            if (0 != stdinArgv.length) {
+                parse(stdinArgv, true, dryRun);
+            }
+        }
+
         return arg;
     }
 
+    private String[] readStdinAsArgv() throws IOException {
+        List<String> stdinArgvList = new ArrayList<String>();
+
+        BufferedReader in = UTF8.bufferedReader(System.in);
+        while (true) {
+            String s = in.readLine();
+            if (s == null) {
+                break;
+            }
+            if (s.trim().isEmpty()) {
+                break;
+            }
```

Comment:
SpotBugs depends on commons-lang3, so we can shorten these lines like below:

```suggestion
            if (null == org.apache.commons.lang3.StringUtils.trimToNull(s)) {
                break;
            }
```

Reply:
Done.